packages/edit-site/src/components/page-patterns/style.scss

```diff
@@ -1,8 +1,19 @@
 .edit-site-patterns {
-	border: 1px solid $gray-800;
+	border-left: 1px solid $gray-800;
```

Comment:
I could not get the save button and pagination height to match exactly when there is a 1px border across the entire pattern canvas. Therefore, I applied the border only to the left side.